app/ExposureHistoryContext.tsx

```diff
@@ -92,17 +31,18 @@ interface ExposureHistoryProps {
 const ExposureHistoryProvider = ({
   children,
 }: ExposureHistoryProps): JSX.Element => {
-  const [hasBeenExposed, setHasBeenExposed] = useState<boolean>(false);
+  const [exposureHistory, setExposureHistory] = useState<ExposureHistory>(
+    blankHistory(),
+  );
   const [userHasNewExposure, setUserHasNewExposure] = useState<boolean>(true);
-  const toggleHasExposure = () => {
-    setHasBeenExposed(!hasBeenExposed);
-  };
 
   useEffect(() => {
     if (!isGPS) {
-      const subscription = startListening((exposures: Possible[]) => {
-        console.log(exposures);
-      });
+      const subscription = BTNativeModule.subscribeToExposureEvents(
```

Comment:
Idk if it makes sense to do something similar for the GPS side of things, but if so we could just pass this subscription handler in on the mounting of the component and pull out the `!isGPS`